nss-cohort-40/bangazon-ecommerce-client-lumberjacks-front
# El new product

# Description
Added functionality to create a new product listing. Please make sure you pull Brian's back-end branch for creating a product listing as well.
 Fixes # (issue)
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions
 Once both branches have been pulled down, click the Sell Product tab in the navbar to ensure the link works. If the link works, it will take you to `/sell`, at which point you will be able to enter details for a new product. Please make sure that you are able to fill out all forms and submit a new product, as well as testing to make sure that all boxes must be filled before the product can be submitted.
 # Checklist:
